src/index.html

```diff
@@ -4,9 +4,502 @@
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="MOYO_logo_img">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="card_container">
+        <div class="card" data-qa="card">
+          <div class="card__product_image">
+            <img
+              class="card__product_image_img"
+              src="./images/imac.jpeg"
+              alt="Моноблок APPLE A1419 iMac 27"
+            >
+          </div>
+          <div class="card__product_name">
+            <h1 class="card__product_name_value">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h1>
+          </div>
+          <div class="card__product_code">
+            <span>
+              Код товара: 195434
+            </span>
+          </div>
+          <div class="card__product_review">
+            <div class="card__review_stars">
+              <div class="card__review_stars_star card__review_stars_star--active"></div>
+              <div class="card__review_stars_star card__review_stars_star--active"></div>
+              <div class="card__review_stars_star card__review_stars_star--active"></div>
+              <div class="card__review_stars_star card__review_stars_star--active"></div>
+              <div class="card__review_stars_star"></div>
```

Comment:
use CSS and BEM modifier to determine active stars

```suggestion
            <div class="card__review_stars card__review_stars---4">
              <div class="card__review_stars_star"></div>
              <div class="card__review_stars_star"></div>
              <div class="card__review_stars_star"></div>
              <div class="card__review_stars_star"></div>
              <div class="card__review_stars_star"></div>
```